[2024-07-03 00:01:30 2197] [DEBUG] [27] 盘库总重量上传成功：weight = 10525
[2024-07-03 00:01:30 2789] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 00:06:30 3803] [DEBUG] [27] 盘库总重量上传成功：weight = 10525
[2024-07-03 00:06:30 4459] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 00:11:30 5463] [DEBUG] [27] 盘库总重量上传成功：weight = 10524
[2024-07-03 00:11:30 6109] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 00:16:30 7201] [DEBUG] [27] 盘库总重量上传成功：weight = 10526
[2024-07-03 00:16:30 7933] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 00:21:30 8872] [DEBUG] [27] 盘库总重量上传成功：weight = 10525
[2024-07-03 00:21:30 9502] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 00:26:31 0535] [DEBUG] [27] 盘库总重量上传成功：weight = 10525
[2024-07-03 00:26:31 1176] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 00:31:31 2085] [DEBUG] [27] 盘库总重量上传成功：weight = 10523
[2024-07-03 00:31:31 2668] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 00:36:31 3638] [DEBUG] [27] 盘库总重量上传成功：weight = 10523
[2024-07-03 00:36:31 4242] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 00:41:31 5258] [DEBUG] [27] 盘库总重量上传成功：weight = 10524
[2024-07-03 00:41:31 5887] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 00:46:31 6876] [DEBUG] [27] 盘库总重量上传成功：weight = 10524
[2024-07-03 00:46:31 7487] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 00:51:31 8444] [DEBUG] [27] 盘库总重量上传成功：weight = 10523
[2024-07-03 00:51:31 9049] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 00:56:32 0067] [DEBUG] [27] 盘库总重量上传成功：weight = 10525
[2024-07-03 00:56:32 0696] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 01:01:32 1757] [DEBUG] [27] 盘库总重量上传成功：weight = 10524
[2024-07-03 01:01:32 2395] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 01:06:32 3349] [DEBUG] [27] 盘库总重量上传成功：weight = 10522
[2024-07-03 01:06:32 3959] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 01:11:32 5116] [DEBUG] [27] 盘库总重量上传成功：weight = 10520
[2024-07-03 01:11:32 5793] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 01:16:32 6891] [DEBUG] [27] 盘库总重量上传成功：weight = 10523
[2024-07-03 01:16:32 7564] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 01:21:32 8569] [DEBUG] [27] 盘库总重量上传成功：weight = 10520
[2024-07-03 01:21:32 9167] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 01:26:33 0083] [DEBUG] [27] 盘库总重量上传成功：weight = 10522
[2024-07-03 01:26:33 0653] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 01:31:33 1557] [DEBUG] [27] 盘库总重量上传成功：weight = 10518
[2024-07-03 01:31:33 2166] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 01:36:33 3263] [DEBUG] [27] 盘库总重量上传成功：weight = 10517
[2024-07-03 01:36:33 3949] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 01:41:33 4822] [DEBUG] [27] 盘库总重量上传成功：weight = 10517
[2024-07-03 01:41:33 5426] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 01:46:33 6534] [DEBUG] [27] 盘库总重量上传成功：weight = 10516
[2024-07-03 01:46:33 7221] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 01:51:33 8168] [DEBUG] [27] 盘库总重量上传成功：weight = 10515
[2024-07-03 01:51:33 8787] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 01:56:33 9836] [DEBUG] [27] 盘库总重量上传成功：weight = 10515
[2024-07-03 01:56:34 0524] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 02:01:34 1490] [DEBUG] [27] 盘库总重量上传成功：weight = 10515
[2024-07-03 02:01:34 2097] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 02:06:34 3175] [DEBUG] [27] 盘库总重量上传成功：weight = 10515
[2024-07-03 02:06:34 3794] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 02:11:34 4953] [DEBUG] [27] 盘库总重量上传成功：weight = 10515
[2024-07-03 02:11:34 5560] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 02:16:34 6594] [DEBUG] [27] 盘库总重量上传成功：weight = 10516
[2024-07-03 02:16:34 7213] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 02:21:34 8074] [DEBUG] [27] 盘库总重量上传成功：weight = 10513
[2024-07-03 02:21:34 8634] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 02:26:34 9571] [DEBUG] [27] 盘库总重量上传成功：weight = 10513
[2024-07-03 02:26:35 0173] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 02:31:35 1076] [DEBUG] [27] 盘库总重量上传成功：weight = 10514
[2024-07-03 02:31:35 1650] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 02:36:35 2584] [DEBUG] [27] 盘库总重量上传成功：weight = 10513
[2024-07-03 02:36:35 3149] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 02:41:35 4209] [DEBUG] [27] 盘库总重量上传成功：weight = 10513
[2024-07-03 02:41:35 4837] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 02:46:35 5863] [DEBUG] [27] 盘库总重量上传成功：weight = 10513
[2024-07-03 02:46:35 6490] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 02:51:35 7490] [DEBUG] [27] 盘库总重量上传成功：weight = 10513
[2024-07-03 02:51:35 8128] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 02:56:35 9280] [DEBUG] [27] 盘库总重量上传成功：weight = 10513
[2024-07-03 02:56:35 9953] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 03:01:36 0942] [DEBUG] [27] 盘库总重量上传成功：weight = 10512
[2024-07-03 03:01:36 1528] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 03:06:36 2651] [DEBUG] [27] 盘库总重量上传成功：weight = 10511
[2024-07-03 03:06:36 3351] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 03:11:36 4197] [DEBUG] [27] 盘库总重量上传成功：weight = 10512
[2024-07-03 03:11:36 4755] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 03:16:36 5685] [DEBUG] [27] 盘库总重量上传成功：weight = 10511
[2024-07-03 03:16:36 6291] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 03:21:36 7319] [DEBUG] [27] 盘库总重量上传成功：weight = 10512
[2024-07-03 03:21:36 7913] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 03:26:36 8916] [DEBUG] [27] 盘库总重量上传成功：weight = 10511
[2024-07-03 03:26:36 9586] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 03:31:37 0579] [DEBUG] [27] 盘库总重量上传成功：weight = 10511
[2024-07-03 03:31:37 1205] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 03:36:37 2233] [DEBUG] [27] 盘库总重量上传成功：weight = 10511
[2024-07-03 03:36:37 2950] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 03:41:37 3992] [DEBUG] [27] 盘库总重量上传成功：weight = 10510
[2024-07-03 03:41:37 4658] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 03:46:37 5655] [DEBUG] [27] 盘库总重量上传成功：weight = 10509
[2024-07-03 03:46:37 6242] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 03:51:37 7232] [DEBUG] [27] 盘库总重量上传成功：weight = 10510
[2024-07-03 03:51:37 7895] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 03:56:37 8821] [DEBUG] [27] 盘库总重量上传成功：weight = 10508
[2024-07-03 03:56:37 9443] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 04:01:38 0442] [DEBUG] [27] 盘库总重量上传成功：weight = 10507
[2024-07-03 04:01:38 1083] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 04:06:38 1938] [DEBUG] [27] 盘库总重量上传成功：weight = 10509
[2024-07-03 04:06:38 2527] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 04:11:38 3753] [DEBUG] [27] 盘库总重量上传成功：weight = 10510
[2024-07-03 04:11:38 4377] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 04:16:38 5358] [DEBUG] [27] 盘库总重量上传成功：weight = 10509
[2024-07-03 04:16:38 5997] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 04:21:38 6987] [DEBUG] [27] 盘库总重量上传成功：weight = 10509
[2024-07-03 04:21:38 7610] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 04:26:38 8703] [DEBUG] [27] 盘库总重量上传成功：weight = 10509
[2024-07-03 04:26:38 9405] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 04:31:39 0437] [DEBUG] [27] 盘库总重量上传成功：weight = 10509
[2024-07-03 04:31:39 1049] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 04:36:39 1966] [DEBUG] [27] 盘库总重量上传成功：weight = 10508
[2024-07-03 04:36:39 2581] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 04:41:39 3590] [DEBUG] [27] 盘库总重量上传成功：weight = 10508
[2024-07-03 04:41:39 4215] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 04:46:39 5176] [DEBUG] [27] 盘库总重量上传成功：weight = 10507
[2024-07-03 04:46:39 5754] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 04:51:39 6649] [DEBUG] [27] 盘库总重量上传成功：weight = 10507
[2024-07-03 04:51:39 7269] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 04:56:39 8320] [DEBUG] [27] 盘库总重量上传成功：weight = 10507
[2024-07-03 04:56:39 8979] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 05:01:40 0009] [DEBUG] [27] 盘库总重量上传成功：weight = 10506
[2024-07-03 05:01:40 0616] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 05:06:40 1501] [DEBUG] [27] 盘库总重量上传成功：weight = 10507
[2024-07-03 05:06:40 2095] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 05:11:40 3101] [DEBUG] [27] 盘库总重量上传成功：weight = 10507
[2024-07-03 05:11:40 3733] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 05:16:40 4765] [DEBUG] [27] 盘库总重量上传成功：weight = 10506
[2024-07-03 05:16:40 5450] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 05:21:40 6449] [DEBUG] [27] 盘库总重量上传成功：weight = 10508
[2024-07-03 05:21:40 7065] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 05:26:40 7995] [DEBUG] [27] 盘库总重量上传成功：weight = 10507
[2024-07-03 05:26:40 8615] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 05:31:40 9626] [DEBUG] [27] 盘库总重量上传成功：weight = 10506
[2024-07-03 05:31:41 0252] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 05:36:41 1242] [DEBUG] [27] 盘库总重量上传成功：weight = 10507
[2024-07-03 05:36:41 1843] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 05:41:41 2922] [DEBUG] [27] 盘库总重量上传成功：weight = 10505
[2024-07-03 05:41:41 3561] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 05:46:41 4505] [DEBUG] [27] 盘库总重量上传成功：weight = 10505
[2024-07-03 05:46:41 5113] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 05:51:41 6036] [DEBUG] [27] 盘库总重量上传成功：weight = 10505
[2024-07-03 05:51:41 6648] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 05:56:41 7553] [DEBUG] [27] 盘库总重量上传成功：weight = 10504
[2024-07-03 05:56:41 8159] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 06:01:41 9041] [DEBUG] [27] 盘库总重量上传成功：weight = 10503
[2024-07-03 06:01:41 9605] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 06:06:42 0748] [DEBUG] [27] 盘库总重量上传成功：weight = 10503
[2024-07-03 06:06:42 1444] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 06:11:42 2386] [DEBUG] [27] 盘库总重量上传成功：weight = 10503
[2024-07-03 06:11:42 3004] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 06:16:42 4016] [DEBUG] [27] 盘库总重量上传成功：weight = 10502
[2024-07-03 06:16:42 4676] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 06:21:42 5754] [DEBUG] [27] 盘库总重量上传成功：weight = 10503
[2024-07-03 06:21:42 6464] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 06:26:42 7410] [DEBUG] [27] 盘库总重量上传成功：weight = 10504
[2024-07-03 06:26:42 8004] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 06:31:42 8951] [DEBUG] [27] 盘库总重量上传成功：weight = 10507
[2024-07-03 06:31:42 9540] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 06:36:43 0589] [DEBUG] [27] 盘库总重量上传成功：weight = 10506
[2024-07-03 06:36:43 1262] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 06:41:43 2219] [DEBUG] [27] 盘库总重量上传成功：weight = 10507
[2024-07-03 06:41:43 2840] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 06:46:43 3809] [DEBUG] [27] 盘库总重量上传成功：weight = 10510
[2024-07-03 06:46:43 4420] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 06:51:43 5470] [DEBUG] [27] 盘库总重量上传成功：weight = 10509
[2024-07-03 06:51:43 6122] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 06:56:43 7117] [DEBUG] [27] 盘库总重量上传成功：weight = 10510
[2024-07-03 06:56:43 7733] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 07:01:43 8584] [DEBUG] [27] 盘库总重量上传成功：weight = 10510
[2024-07-03 07:01:43 9132] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 07:06:44 0104] [DEBUG] [27] 盘库总重量上传成功：weight = 10514
[2024-07-03 07:06:44 0673] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 07:11:44 1551] [DEBUG] [27] 盘库总重量上传成功：weight = 10512
[2024-07-03 07:11:44 2135] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 07:16:44 3000] [DEBUG] [27] 盘库总重量上传成功：weight = 10513
[2024-07-03 07:16:44 3594] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 07:21:44 4497] [DEBUG] [27] 盘库总重量上传成功：weight = 10513
[2024-07-03 07:21:44 5133] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 07:26:44 6028] [DEBUG] [27] 盘库总重量上传成功：weight = 10513
[2024-07-03 07:26:44 6629] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 07:31:44 7594] [DEBUG] [27] 盘库总重量上传成功：weight = 10517
[2024-07-03 07:31:44 8217] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 07:36:44 9242] [DEBUG] [27] 盘库总重量上传成功：weight = 10527
[2024-07-03 07:36:44 9877] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 07:41:45 0730] [DEBUG] [27] 盘库总重量上传成功：weight = 10527
[2024-07-03 07:41:45 1315] [DEBUG] [27] 盘库数据上传成功!
                                                                                                                                                         [2024-07-03 07:48:37 2190] [DEBUG] [6] 料仓skuid读取完毕
[2024-07-03 07:48:37 5309] [ERROR] [6] 连接plc失败！错误码：-1; Socket Connect 192.168.8.10:102 Exception -> 套接字操作尝试一个无法连接的主机。 192.168.8.10:102
[2024-07-03 07:48:37 6693] [DEBUG] [6] 采集器初始化完成
[2024-07-03 07:48:38 1401] [INFO] [1] 数据库清理已完成，清理数量： t_info_record = 0t_price_info_record = 0t_power_record = 0
[2024-07-03 07:48:38 4983] [ERROR] [6] Error: token获取异常  System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/User/innerLogin ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 套接字操作尝试一个无法连接的主机。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.GetToken() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 280
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/User/innerLogin ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 套接字操作尝试一个无法连接的主机。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 07:48:38 4983] [DEBUG] [6] 采集器打开成功
[2024-07-03 07:48:38 5003] [INFO] [6] MB从站启动成功
[2024-07-03 07:48:38 5118] [INFO] [24] 开启数据采集
[2024-07-03 07:48:41 3564] [ERROR] [24] GX1 : Socket Connect 192.168.8.20:502 Exception -> 套接字操作尝试一个无法连接的主机。 192.168.8.20:502
[2024-07-03 07:48:41 5900] [ERROR] [24] PLC1 : Socket Connect 192.168.8.10:102 Exception -> 套接字操作尝试一个无法连接的主机。 192.168.8.10:102
[2024-07-03 07:48:45 4386] [ERROR] [24] GX1 : Socket Connect 192.168.8.20:502 Exception -> 套接字操作尝试一个无法连接的主机。 192.168.8.20:502
[2024-07-03 07:48:45 6556] [ERROR] [24] PLC1 : Socket Connect 192.168.8.10:102 Exception -> 套接字操作尝试一个无法连接的主机。 192.168.8.10:102
[2024-07-03 07:48:48 5070] [ERROR] [6] Error: token获取异常  System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/User/innerLogin ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 套接字操作尝试一个无法连接的主机。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.GetToken() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 280
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/User/innerLogin ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 套接字操作尝试一个无法连接的主机。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 07:48:49 9496] [ERROR] [24] GX1 : 连接 192.168.8.20:502 失败，超时时间为 100 ms
[2024-07-03 07:48:54 1966] [ERROR] [24] GX1 : 连接 192.168.8.20:502 失败，超时时间为 100 ms
[2024-07-03 07:48:58 2249] [ERROR] [24] GX1 : 连接 192.168.8.20:502 失败，超时时间为 100 ms
[2024-07-03 07:48:58 7445] [INFO] [6] 开启上传任务！
[2024-07-03 07:49:02 6869] [ERROR] [24] GX1 : 连接 192.168.8.20:502 失败，超时时间为 100 ms
[2024-07-03 07:49:07 1490] [ERROR] [24] GX1 : 连接 192.168.8.20:502 失败，超时时间为 100 ms
[2024-07-03 07:49:08 7143] [DEBUG] [27] 盘库总重量上传成功：weight = 10524
[2024-07-03 07:49:08 7973] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 07:49:11 6043] [ERROR] [24] GX1 : 连接 192.168.8.20:502 失败，超时时间为 100 ms
[2024-07-03 07:54:08 8798] [DEBUG] [27] 盘库总重量上传成功：weight = 10522
[2024-07-03 07:54:08 9393] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 07:59:09 0387] [DEBUG] [27] 盘库总重量上传成功：weight = 10525
[2024-07-03 07:59:09 1400] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 08:04:09 2415] [DEBUG] [27] 盘库总重量上传成功：weight = 10498
[2024-07-03 08:04:09 3023] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 08:09:09 3921] [DEBUG] [27] 盘库总重量上传成功：weight = 10617
[2024-07-03 08:09:09 4529] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 08:14:09 5567] [DEBUG] [27] 盘库总重量上传成功：weight = 10773
[2024-07-03 08:14:09 6230] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 08:19:09 7160] [DEBUG] [27] 盘库总重量上传成功：weight = 10945
[2024-07-03 08:19:09 7788] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 08:24:09 8793] [DEBUG] [27] 盘库总重量上传成功：weight = 10994
[2024-07-03 08:24:09 9383] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 08:29:10 0225] [DEBUG] [27] 盘库总重量上传成功：weight = 11046
[2024-07-03 08:29:10 0814] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 08:34:10 1754] [DEBUG] [27] 盘库总重量上传成功：weight = 10935
[2024-07-03 08:34:10 2405] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 08:39:10 3336] [DEBUG] [27] 盘库总重量上传成功：weight = 10801
[2024-07-03 08:39:10 3906] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 08:44:10 4867] [DEBUG] [27] 盘库总重量上传成功：weight = 10800
[2024-07-03 08:44:10 5497] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 08:49:10 6443] [DEBUG] [27] 盘库总重量上传成功：weight = 10730
[2024-07-03 08:49:10 7031] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 08:54:10 8097] [DEBUG] [27] 盘库总重量上传成功：weight = 10844
[2024-07-03 08:54:10 8826] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 08:59:10 9672] [DEBUG] [27] 盘库总重量上传成功：weight = 10788
[2024-07-03 08:59:11 0235] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 09:04:11 1247] [DEBUG] [27] 盘库总重量上传成功：weight = 10999
[2024-07-03 09:04:11 1864] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 09:09:11 2792] [DEBUG] [27] 盘库总重量上传成功：weight = 10992
[2024-07-03 09:09:11 3390] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 09:14:11 4346] [DEBUG] [27] 盘库总重量上传成功：weight = 10798
[2024-07-03 09:14:11 4999] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 09:19:11 6346] [DEBUG] [27] 盘库总重量上传成功：weight = 10793
[2024-07-03 09:19:11 7017] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 09:24:11 7967] [DEBUG] [27] 盘库总重量上传成功：weight = 10653
[2024-07-03 09:24:11 8604] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 09:29:11 9538] [DEBUG] [27] 盘库总重量上传成功：weight = 10696
[2024-07-03 09:29:12 0345] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 09:34:12 1357] [DEBUG] [27] 盘库总重量上传成功：weight = 10645
[2024-07-03 09:34:12 1954] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 09:39:12 2775] [DEBUG] [27] 盘库总重量上传成功：weight = 10878
[2024-07-03 09:39:12 3319] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 09:44:12 4314] [DEBUG] [27] 盘库总重量上传成功：weight = 11030
[2024-07-03 09:44:12 4943] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 09:49:12 5968] [DEBUG] [27] 盘库总重量上传成功：weight = 10930
[2024-07-03 09:49:12 6583] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 09:54:12 7438] [DEBUG] [27] 盘库总重量上传成功：weight = 10706
[2024-07-03 09:54:12 8057] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 09:59:12 9255] [DEBUG] [27] 盘库总重量上传成功：weight = 10551
[2024-07-03 09:59:12 9936] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 10:04:13 0818] [DEBUG] [27] 盘库总重量上传成功：weight = 10550
[2024-07-03 10:04:13 1450] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 10:09:13 2429] [DEBUG] [27] 盘库总重量上传成功：weight = 10312
[2024-07-03 10:09:13 3092] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 10:14:13 4071] [DEBUG] [27] 盘库总重量上传成功：weight = 10199
[2024-07-03 10:14:13 4701] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 10:19:13 5732] [DEBUG] [27] 盘库总重量上传成功：weight = 10212
[2024-07-03 10:19:13 6355] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 10:24:13 7184] [DEBUG] [27] 盘库总重量上传成功：weight = 10358
[2024-07-03 10:24:13 7751] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 10:29:13 8688] [DEBUG] [27] 盘库总重量上传成功：weight = 10435
[2024-07-03 10:29:13 9342] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 10:34:14 0185] [DEBUG] [27] 盘库总重量上传成功：weight = 10599
[2024-07-03 10:34:14 0783] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 10:39:14 1761] [DEBUG] [27] 盘库总重量上传成功：weight = 10725
[2024-07-03 10:39:14 2359] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 10:44:14 3287] [DEBUG] [27] 盘库总重量上传成功：weight = 10464
[2024-07-03 10:44:14 3878] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 10:49:14 4903] [DEBUG] [27] 盘库总重量上传成功：weight = 10253
[2024-07-03 10:49:14 5572] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 10:54:14 6483] [DEBUG] [27] 盘库总重量上传成功：weight = 9879
[2024-07-03 10:54:14 7077] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 10:59:14 7952] [DEBUG] [27] 盘库总重量上传成功：weight = 9859
[2024-07-03 10:59:14 8554] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 11:04:14 9458] [DEBUG] [27] 盘库总重量上传成功：weight = 9759
[2024-07-03 11:04:15 0055] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 11:09:15 1035] [DEBUG] [27] 盘库总重量上传成功：weight = 9767
[2024-07-03 11:09:15 1660] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 11:14:15 2632] [DEBUG] [27] 盘库总重量上传成功：weight = 9539
[2024-07-03 11:14:15 3235] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 11:19:15 4490] [DEBUG] [27] 盘库总重量上传成功：weight = 9524
[2024-07-03 11:19:15 5107] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 11:24:15 5892] [DEBUG] [27] 盘库总重量上传成功：weight = 9625
[2024-07-03 11:24:15 6447] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 11:29:15 7290] [DEBUG] [27] 盘库总重量上传成功：weight = 9590
[2024-07-03 11:29:15 7880] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 11:34:15 8734] [DEBUG] [27] 盘库总重量上传成功：weight = 9573
[2024-07-03 11:34:15 9342] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 11:39:16 0440] [DEBUG] [27] 盘库总重量上传成功：weight = 9608
[2024-07-03 11:39:16 1148] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 11:44:16 2135] [DEBUG] [27] 盘库总重量上传成功：weight = 9639
[2024-07-03 11:44:16 2738] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 11:49:16 3683] [DEBUG] [27] 盘库总重量上传成功：weight = 9659
[2024-07-03 11:49:16 4286] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 11:54:16 5168] [DEBUG] [27] 盘库总重量上传成功：weight = 9640
[2024-07-03 11:54:16 5787] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 11:59:16 6847] [DEBUG] [27] 盘库总重量上传成功：weight = 9711
[2024-07-03 11:59:16 7526] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 12:04:16 8461] [DEBUG] [27] 盘库总重量上传成功：weight = 9744
[2024-07-03 12:04:16 9119] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 12:09:17 0140] [DEBUG] [27] 盘库总重量上传成功：weight = 9664
[2024-07-03 12:09:17 0748] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 12:14:17 1862] [DEBUG] [27] 盘库总重量上传成功：weight = 9629
[2024-07-03 12:14:17 2540] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 12:19:17 3503] [DEBUG] [27] 盘库总重量上传成功：weight = 9667
[2024-07-03 12:19:17 4111] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 12:24:17 5026] [DEBUG] [27] 盘库总重量上传成功：weight = 9732
[2024-07-03 12:24:17 5665] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 12:29:17 6581] [DEBUG] [27] 盘库总重量上传成功：weight = 9734
[2024-07-03 12:29:17 7200] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 12:34:17 8157] [DEBUG] [27] 盘库总重量上传成功：weight = 9739
[2024-07-03 12:34:17 8742] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 12:39:17 9742] [DEBUG] [27] 盘库总重量上传成功：weight = 9775
[2024-07-03 12:39:18 0413] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 12:44:18 1321] [DEBUG] [27] 盘库总重量上传成功：weight = 9755
[2024-07-03 12:44:18 1925] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 12:49:18 2931] [DEBUG] [27] 盘库总重量上传成功：weight = 9807
[2024-07-03 12:49:18 3585] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 12:54:18 4609] [DEBUG] [27] 盘库总重量上传成功：weight = 9837
[2024-07-03 12:54:18 5187] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 12:59:18 6053] [DEBUG] [27] 盘库总重量上传成功：weight = 9839
[2024-07-03 12:59:18 6649] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 13:04:18 7646] [DEBUG] [27] 盘库总重量上传成功：weight = 9717
[2024-07-03 13:04:18 8306] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 13:09:18 9251] [DEBUG] [27] 盘库总重量上传成功：weight = 9715
[2024-07-03 13:09:18 9874] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 13:14:19 0932] [DEBUG] [27] 盘库总重量上传成功：weight = 9687
[2024-07-03 13:14:19 1628] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 13:19:19 2650] [DEBUG] [27] 盘库总重量上传成功：weight = 9652
[2024-07-03 13:19:19 3322] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 13:24:19 4353] [DEBUG] [27] 盘库总重量上传成功：weight = 9583
[2024-07-03 13:24:19 5004] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 13:29:19 7931] [DEBUG] [27] 盘库总重量上传成功：weight = 9659
[2024-07-03 13:29:19 8559] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 13:34:19 9620] [DEBUG] [27] 盘库总重量上传成功：weight = 9731
[2024-07-03 13:34:20 0278] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 13:39:20 1290] [DEBUG] [27] 盘库总重量上传成功：weight = 9744
[2024-07-03 13:39:20 1961] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 13:44:20 2945] [DEBUG] [27] 盘库总重量上传成功：weight = 9969
[2024-07-03 13:44:20 3583] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 13:49:20 4444] [DEBUG] [27] 盘库总重量上传成功：weight = 9958
[2024-07-03 13:49:20 5027] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 13:54:20 6105] [DEBUG] [27] 盘库总重量上传成功：weight = 9865
[2024-07-03 13:54:20 6789] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 13:59:20 7793] [DEBUG] [27] 盘库总重量上传成功：weight = 9965
[2024-07-03 13:59:20 8412] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 14:04:20 9294] [DEBUG] [27] 盘库总重量上传成功：weight = 9923
[2024-07-03 14:04:20 9876] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 14:09:21 0950] [DEBUG] [27] 盘库总重量上传成功：weight = 9846
[2024-07-03 14:09:21 1628] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 14:14:21 2665] [DEBUG] [27] 盘库总重量上传成功：weight = 9777
[2024-07-03 14:14:21 3337] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 14:19:21 4237] [DEBUG] [27] 盘库总重量上传成功：weight = 9900
[2024-07-03 14:19:21 4869] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 14:24:21 5951] [DEBUG] [27] 盘库总重量上传成功：weight = 9967
[2024-07-03 14:24:21 6658] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 14:29:21 7523] [DEBUG] [27] 盘库总重量上传成功：weight = 9973
[2024-07-03 14:29:21 8132] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 14:34:21 9134] [DEBUG] [27] 盘库总重量上传成功：weight = 10071
[2024-07-03 14:34:21 9768] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 14:39:22 0675] [DEBUG] [27] 盘库总重量上传成功：weight = 10073
[2024-07-03 14:39:22 1314] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 14:44:22 2159] [DEBUG] [27] 盘库总重量上传成功：weight = 10100
[2024-07-03 14:44:22 2742] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 14:49:22 3655] [DEBUG] [27] 盘库总重量上传成功：weight = 10069
[2024-07-03 14:49:22 4289] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 14:54:22 5354] [DEBUG] [27] 盘库总重量上传成功：weight = 10205
[2024-07-03 14:54:22 6018] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 14:59:22 6903] [DEBUG] [27] 盘库总重量上传成功：weight = 10325
[2024-07-03 14:59:22 7542] [DEBUG] [27] 盘库数据上传成功!
[2024-07-03 15:04:43 8137] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 15:05:04 8478] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 15:10:25 8932] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 15:10:46 9360] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 15:16:07 9783] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 15:16:29 0086] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 15:21:50 0687] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 15:22:11 1095] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 15:27:32 1466] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 15:27:53 1796] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 15:33:14 2253] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 15:33:35 2553] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 15:38:56 2915] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 15:39:17 3491] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 15:44:38 4007] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 15:44:59 4320] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 15:50:20 4822] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 15:50:41 5322] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 15:56:02 5837] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 15:56:23 6272] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 16:01:44 6832] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 16:02:05 7302] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 16:07:26 7655] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 16:07:47 8021] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 16:13:08 8572] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 16:13:29 8934] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 16:18:50 9471] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 16:19:11 9920] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 16:24:33 0357] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 16:24:54 0781] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 16:30:15 1267] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 16:30:36 1962] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 16:35:57 2560] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 16:36:18 3028] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 16:41:39 3524] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 16:42:00 4035] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 16:47:21 4559] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 16:47:42 5080] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 16:53:03 5558] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 16:53:24 6073] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 16:58:45 6631] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 16:59:06 7171] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 17:04:27 7607] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 17:04:48 7884] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 17:10:09 8260] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 17:10:30 8689] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 17:15:51 9313] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 17:16:12 9749] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 17:21:34 0217] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 17:21:55 0673] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 17:27:16 0999] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 17:27:37 1105] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 17:32:58 1419] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 17:33:19 1897] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 17:38:40 2372] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 17:39:01 2762] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 17:44:22 3386] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 17:44:43 3772] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 17:50:04 4121] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 17:50:25 4623] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 17:55:46 5091] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 17:56:07 5452] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

[2024-07-03 18:01:28 5836] [ERROR] [27] 料仓实时总重量上传失败！Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/LibraryInfo/InsertLibrary
[2024-07-03 18:01:49 6138] [ERROR] [27] 料仓实时数据上传失败！System.AggregateException: 发生一个或多个错误。 ---> Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   在 IOT.Serve.Upload.UpLodeDataOfAllBins() 位置 D:\work\IOT_Price\IOT_Price\IOT_Price\Serve\Upload.cs:行号 639
---> (内部异常 #0) Flurl.Http.FlurlHttpException: Call failed. 发送请求时出错。: POST http://47.114.126.3:9993/api/SiloInfo/Add ---> System.Net.Http.HttpRequestException: 发送请求时出错。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 47.114.126.3:9993
   在 System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   在 System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   在 System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Flurl.Http.FlurlRequest.<HandleExceptionAsync>d__34.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Flurl.Http.FlurlRequest.<SendAsync>d__29.MoveNext()<---

